app/models/rateitapp/rating.rb

```diff
@@ -19,5 +19,12 @@ class Rating < ActiveRecord::Base
     validates :ratable_type, presence: true
     validates :ratable_id, presence: true
     validates :user_id, presence: true
+    validate :value_is_within_range
+
+    def value_is_within_range
+      if (value && ratable_type) && !(0..5).cover?(value)
+        errors.add(:value, "can't be outside range")
+      end
+    end
```

Comment:
I think we would just delete the rating instead of marking it nil. I can't think of any value right now for the concept of a non-rating other than its lack of existence. Otherwise we'll end up needing a method to retrieve zombie ratings, which is entertaining but not useful. 😄